### ISSUE TITLE
[Reqest Doc] wiring

### ISSUE THREAD

**Issue by et-ness** [gentilkiwi/st25tb_kameleon_piko#1]
Could you update README or make a new file with wiring between Pi PICO  and module TRF7970A?

**Comment by gentilkiwi** [comment]
The README is work-in-progress.
All kameleon boards (including piko) are here: https://oshwlab.com/gentilkiwi/st25tb_kameleon

**Comment by et-ness** [comment]
are you thinking of developing a flipper zero board?
Flipper zero allow communication with spi on exported GPIO and has some pins for EN and IRQ.

**Comment by gentilkiwi** [comment]
:) No, the Flipper Zero and Proxmark3 are already best for all use cases (and the TRF7970A is **very** limited)

**Comment by et-ness** [comment]
Yes is very limited, I hate this chip, but is the best solution which we have for 14b emulation.
For years I have looked for an alternative but there has never been one.
I wasted many hours trying to emulate a 14b tag without ever being successful, I had learned the datasheet by heart, the registers and the default values.
When I found your project I didn't want it to be possible, I was convinced that TRF7970A was not capable of emulating a 14b tag.
